packages/whale-api-client/src/api/poolpairs.ts

```diff
@@ -172,6 +172,10 @@ export interface PoolSwapData {
    * To handle for optional value as Whale service might fail to resolve when indexing
    */
   to?: PoolSwapFromToData
+  /**
+   * To handle for optional value as Whale service might fail to resolve when indexing
+   */
+  isSell?: boolean
```

Comment:
```suggestion
  type: 'sell' | 'buy'
```

Should we use enum instead for better evolutionary of this field?